tests/functional/views/basic_lti_launch_test.py

```diff
@@ -87,91 +55,31 @@ def test_basic_lti_launch(
             urlencode({"url": "https://url-configured.com/document.pdf"})
             in js_config["viaUrl"]
         )
+        assert (
```

Comment:
Adding this assertion about the side effect of the launch. It will register an assignment in our DB.